tests/test/test.js

```diff
@@ -463,6 +523,37 @@ describe('Importer / Exporter (Roundtrip)', function() {
                 });
             }
 
+            it('roundtrips a morph target animation', function() {
+                let dir = '01_morphed_cube';
+                let outDirPath = path.resolve(OUT_PREFIX, 'roundtrip', dir, outDirName);
+                let gltfPath = path.resolve(outDirPath, dir + '.gltf');
+                const asset = JSON.parse(fs.readFileSync(gltfPath));
+                assert.strictEqual(asset.animations.length, 1);
+
+                const animation = asset.animations[0];
+                assert.strictEqual(animation.channels.length, 1);
+                assert.strictEqual(animation.samplers.length, 1);
+                assert.strictEqual(animation.channels[0].sampler, 0);
+                assert.strictEqual(animation.channels[0].target.path, 'weights');
+                assert.strictEqual(animation.samplers[0].interpolation, 'CUBICSPLINE');
+
+                const animatedNode = asset.nodes[animation.channels[0].target.node];
+                const animatedMesh = asset.meshes[animatedNode.mesh];
+                const targetNames = animatedMesh.extras.targetNames;
+                assert.strictEqual(targetNames.length, 2);
+                assert.notStrictEqual(targetNames.indexOf('Top'), -1);
+                assert.notStrictEqual(targetNames.indexOf('Bottom'), -1);
```

Comment:
Does this test still pass if we expect the indices to be 0 and 1, respectively? I think we'd had a bug in the old exporter with the order of morph targets being unstable.

Reply:
This is just checking `indexOf(...) !== -1`, meaning testing for presence in the list, regardless of order.

Reply:
Right, I guess I mean that if the test _does_ pass with a strict requirement on order, we should probably strictly test the order. Maintaining consistent output order is important, because the only other indication of which morph target is which is the `extras.targetNames` array, which isn't an official spec feature.